scripts/build_cuvslam_in_docker.sh

```
@@ -0,0 +1,63 @@
+
+BUILD_TYPE=$1
+OUTPUT_DIR=${2:-$(pwd)/build_docker}
+OUTPUT_DIR=$(realpath -m "$OUTPUT_DIR")
+mkdir -p "$OUTPUT_DIR"
+
+DOCKERFILE=$(dirname "$(realpath "$0")")/Dockerfile
+DOCKER_BUILD_ARGS=""
+[ -n "${CUDA_VERSION:-}" ] && DOCKER_BUILD_ARGS="$DOCKER_BUILD_ARGS --build-arg CUDA_VERSION=$CUDA_VERSION"
+[ -n "${UBUNTU_VERSION:-}" ] && DOCKER_BUILD_ARGS="$DOCKER_BUILD_ARGS --build-arg UBUNTU_VERSION=$UBUNTU_VERSION"
+[ -n "${BASE_IMAGE:-}" ] && DOCKER_BUILD_ARGS="$DOCKER_BUILD_ARGS --build-arg BASE_IMAGE=$BASE_IMAGE"
+docker build -f "$DOCKERFILE" . --network host $DOCKER_BUILD_ARGS --tag cuvslam:local
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)